arches/app/templates/views/components/widgets/switch.htm

```diff
@@ -3,7 +3,8 @@
 
 {% block form %}
 <div class="toggle-container">
-    <span class="switch switch-small switch-widget" data-bind="css: {'on': getvalue() === true, 'null': getvalue() === null, 'disabled': disabled }, click: setvalue"><small></small></span>
+    <span class="switch switch-small switch-widget" data-bind="css: {'on': getvalue() === true, 'null': getvalue() === null, 'disabled': disabled }, 
+          onEnterkeyClick, onSpaceClick, click: setvalue, attr: {'aria-checked': getvalue}" tabindex="0" role="switch"><small></small></span>
```

Comment:
add `aria-describedby` that points to the subtitle span below

Reply:
I'm experiencing `aria-describedby` breaking the keyboard click handlers and not letting user tab out of the switch element. How do you feel about concatenating the `label` & `localizedSubtitle`?

Reply:
Hmm, that's weird.  I guess that's fine to concatenate those strings.